6.3/main.py

```diff
@@ -0,0 +1,25 @@
+
+#get a sentence
+#return a list of the length of each word in the sentence
+#using list comprehension
+def words_length(sentence):
```

Comment:
Use type hints and return types -docum
https://www.geeksforgeeks.org/duck-typing-in-python/
https://stackoverflow.com/questions/125367/dynamic-type-languages-versus-static-type-languages
https://docs.python.org/3/library/typing.html
https://realpython.com/lessons/pros-and-cons-type-hints/

Do this for all functions.